color.go

```diff
@@ -65,6 +65,29 @@ const (
 	CrossedOut
 )
 
+const (
+	ResetBoldFaint Attribute = iota + 22
+	ResetItalic
+	ResetUnderline
+	ResetBlinkSlow
+	ResetBlinkRapid
+	ResetReverseVideo
+	ResetConcealed
+	ResetCrossedOut
```

Comment:
I think we should call them differently. And I think we only should have the modes here that we can truly represent. Based on [Wikipedia](https://en.wikipedia.org/wiki/ANSI_escape_code):

![CleanShot 2023-10-16 at 17 58 39@2x](https://github.com/fatih/color/assets/438920/1f73d0b9-af92-4576-b95d-e9dafdf86bd8)


We should use these values. I also double checked it in the official ECMA doc (https://www.ecma-international.org/wp-content/uploads/ECMA-48_5th_edition_june_1991.pdf), page 61/62. So let's use these values:

* 22: `ResetBold`
* 23: `ResetItalic`
* 24: `ResetUnderline`
* 25: `ResetBlinking`
* 26:  _ (nothing is used here, it's reserved)
* 27: `ResetReversed`
* 28: `ResetConcealed`
* 29: `ResetCrossedOut`

Reply:
My original idea for the names of the Resets was to try and maintain the original names, simply prefixing them with 'Reset'.
But I will follow your suggestions

Reply:
new test case according to your suggestion:
![image](https://github.com/fatih/color/assets/2430354/bea76d68-0fc7-456f-8533-416edf28b69e)